release/src/test_release.py

```diff
@@ -82,4 +82,24 @@ def test_is_multi():
 def test_is_multi_not():
   project="sonar-dummy"
   buildnumber="359"  
-  assert not is_multi(project,buildnumber)
\ No newline at end of file
+  assert not is_multi(project,buildnumber)
+
+def test_check_public_not():
+  project="sonar-dummy"
+  buildnumber="359"  
+  assert not check_public(project, buildnumber)
+
+def test_check_public():
+  project="sonar-dummy-oss"
+  buildnumber="23"  
+  assert check_public(project, buildnumber)  
+
+def test_distribute_build():
+  project="sonar-dummy-oss"
+  buildnumber="23"  
+  distribute_build(project, buildnumber)  
+
+def test_distribute_build_fail():
+  project="sonar-dummy"
+  buildnumber="359"  
+  distribute_build(project, buildnumber)    
```

Comment:
I think we could assert on the failure message at the end of this test, using `capsys`